src/outlookCalendar/getOutlookEvents.ts

```
@@ -101,6 +101,33 @@ const buildEwsPathname = (url: URL): void => {
+  // If URL already has a non-default port, it's preserved by toString()
+  if (url.port) {
+    return baseUrl;
+  }
+
+  // Check if original URL had an explicit default port that was normalized away
+  // Match port in authority section (after //) or at start (for URLs without protocol)
+  const portMatch = originalUrl.match(/(?:\/\/|^)[^/:]+:(\d+)/);
+  if (portMatch) {
+    const explicitPort = portMatch[1];
+    // Only restore if it's a default port that was dropped
+    if ((url.protocol === 'https:' && explicitPort === '443') ||
```

ESLint

[error] Insert `⏎······`

(prettier/prettier)